.github/workflows/zowe-cli.yml

```diff
@@ -129,11 +129,16 @@ jobs:
       env:
         PATH: ${{ github.workspace }}\__tests__\__resources__\application_instances;${{ env.PATH }}
 
-    - name: Integration Tests
-      id: integration
-      if: ${{ always() && steps.build.outcome == 'success' }}
+    - name: Integration Tests (Native)
+      id: integration-native
+      if: ${{ always() && steps.build.outcome == 'success' && !(github.event.inputs.test-type == 'binary' || github.event_name == 'push') }}
```

Comment:
Is the `|| github.event_name == 'push'` needed ?
I believe this means that it won't run on a branch push ?

Reply:
The goal of this is to get the native tests to run when the Start Daemon step has not run, and get the daemon tests to run when the daemon was built. I just took the conditional logic for the Build Binary if statement and inverted it (or at least tried to). If there's a better suggestion for accomplishing this behavior (or if my changes' behavior is wrong), I appreciate any suggestions.

Reply:
I'm afraid I don't have any good suggestions until we see this workflow in action.
 I just wanted to understand how these conditions would affect the integration test 😋